### ISSUE TITLE
Could not find module `lodash` using v4.17.0 with FastBoot 

### ISSUE THREAD

**Issue by josemarluedke** [mike-north/ember-lodash#94]
We just tried to upgrade to the new version of ember-lodash and we had problem using it when executing in FastBoot. 

The full stack trace error is as follows: 

```
Error: Could not find module `lodash` imported from `athena/instance-initializers/fastboot/thrift`
    at missingModule (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:244:11)
    at findModule (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:255:17)
    at Module.findDeps (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:195:24)
    at findModule (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:258:11)
    at requireModule (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:25:15)
    at resolveInitializer (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:136572:18)
    at registerInstanceInitializers (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:136591:31)
    at exports.default (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:136612:5)
    at Module.callback (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/athena.js:286:41)
    at Module.exports (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:136:32)
    at Module._reify (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:169:59)
    at Module.reify (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:155:27)
    at Module.exports (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:134:10)
    at Object.requireModule (/Users/josemarluedke/Projects/Neighborly/demeter/main/dist/dev/fastboot/vendor.js:28:18)
    at Object.<anonymous> (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/fastboot/dist/cjs/ember-app.js:160:18)
    at VMSandbox.run (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/fastboot/dist/cjs/vm-sandbox.js:18:13)
    at EmberApp.createEmberApp (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/fastboot/dist/cjs/ember-app.js:159:30)
    at EmberApp.retrieveSandboxedApp (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/fastboot/dist/cjs/ember-app.js:184:17)
    at new EmberApp (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/fastboot/dist/cjs/ember-app.js:48:21)
    at FastBoot._buildEmberApp (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/fastboot/dist/cjs/index.js:111:17)
    at new FastBoot (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/fastboot/dist/cjs/index.js:52:10)
    at fastbootExpressMiddleware (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/fastboot-express-middleware/index.js:24:16)
    at exec.then (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/ember-cli-fastboot/lib/tasks/fastboot-server.js:34:71)
    at tryCatch (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/rsvp/dist/lib/rsvp/-internal.js:215:12)
    at invokeCallback (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/rsvp/dist/lib/rsvp/-internal.js:230:13)
    at publish (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/rsvp/dist/lib/rsvp/-internal.js:198:7)
    at flush (/Users/josemarluedke/Projects/Neighborly/demeter/main/node_modules/rsvp/dist/lib/rsvp/asap.js:85:5)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by josemarluedke** [comment]
I think this is related to #89

**Comment by tchak** [comment]
should be fixed with #89 

Feel free to reopen if problem persists
